spec/02-integration/20-wasm/11-missing-filters_spec.lua

```diff
@@ -0,0 +1,87 @@
+local helpers = require "spec.helpers"
+
+local FILTER_PATH = assert(helpers.test_conf.wasm_filters_path)
+
+-- no cassandra support
+for _, strategy in helpers.each_strategy({ "postgres" }) do
```

Comment:
```suggestion
for _, strategy in helpers.each_strategy({ "postgres", "off" }) do
```
I don't remember the order things happen. Isn't it relevant to test dbless mode?

Reply:
Good call! I updated the test coverage to include dbless.

Reply:
It's equivalent but maybe use `helpers.all_strategies()` instead